packages/manager/apps/telecom/src/app/telecom/pack/move/offers/move-offers.controller.js

```diff
@@ -181,14 +181,29 @@ export default class PackMoveOffersCtrl {
     let totalOfferPrice = offer.prices.price.price
       ? offer.prices.price.price.value
       : 0;
+    let optionComfort = false;
 
     offer.options.forEach((option) => {
-      if (option.name === 'gtr_ovh' && option.selected) {
-        totalOfferPrice += option.optionalPrice.value;
+      if (
+        option.name.match(/^gtr_/) &&
```

Comment:
**nitpick:** Some `String.prototype.match()` can be replaced by `String.prototype.startsWith()`.

Reply:
done